collector/lib/NetworkStatusNotifier.cpp

```diff
@@ -208,7 +208,7 @@ void NetworkStatusNotifier::RunSingle(IDuplexClientWriter<sensor::NetworkConnect
   WaitUntilWriterStarted(writer, 10);
 
   ConnMap old_conn_state;
-  ContainerEndpointMap old_cep_state;
+  AdvertisedEndpointMap old_cep_state;
```

Comment:
Perhaps put a comment here explaining why an AdvertisedEndpointMap is used here.

Reply:
This is part of the API of ConnectionTracker API (the return type of the method used to retrieve the sets). I don't think that the API user has to know about internals.